### PR TITLE
feat(condo): DOMA-8408 ticket auto assignment by classifiers

### PR DESCRIPTION
**Pull request by Alllex202**
### 1. The `TicketAutoAssignment` scheme has been added.
It allows you to specify who should be appointed as the executor/assignee of ticket based on a complete match with the classifier.
Potentially you can add dependencies of other fields.

**This currently works on top of the default auto-assignment logic by specialization.**
1) Try to get an employee from the list of rules from `TicketAutoAssignment` (performer/responsible may be empty)
2) If we couldn’t do this, then select an employee depending on their specialization
-------
### 2. Added a page for editing this schema (This is not for all users! Only for those who have direct access to this schema!)
Please don't be strict when reviewing this page 👉👈
I decided to place all the code for this page in one file, because this is not a page for public use.

-------
### 3. Fixed data refetching when using `useAllObjects` hook

-------
_Links:_ 
 - [Architecture](https://docs.google.com/presentation/d/1CpjFD01_BwvXw_ZB6_FQg9VxeamRf8wWn8e90nIw3nI/edit#slide=id.g26a838ad4d6_0_0)

### CODE REVIEW

File: apps/condo/lang/en/en.json [hunk 1]
```diff
@@ -874,6 +874,21 @@
...
+  "pages.ticket.autoAssignment.columns.operations.title": "Operations",
+  "pages.ticket.autoAssignment.button.create.label": "Create",
+  "pages.ticket.autoAssignment.button.cancel.label": "Cancel",
+  "pages.ticket.autoAssignment.button.refresh.label": "Refresh",
+  "pages.ticket.autoAssignment.button.save.label": "Save",
+  "pages.ticket.autoAssignment.button.edit.label": "Edit",
+  "pages.ticket.autoAssignment.button.delete.label": "Delete",
+  "pages.ticket.autoAssignment.notifications.save.message": "Данные сохранены",
+  "pages.ticket.autoAssignment.notifications.delete.message": "Данные удалены",
+  "pages.ticket.autoAssignment.notifications.error.message": "Что-то пошло не так",
```
Comment (nomerdvadcatpyat) [inline apps/condo/lang/en/en.json:891]: ru translations in en file :)

File: apps/condo/lang/ru/ru.json [hunk 2]
```diff
@@ -874,6 +874,21 @@
...
+  "pages.ticket.autoAssignment.columns.operations.title": "Действия",
+  "pages.ticket.autoAssignment.button.create.label": "Создать",
+  "pages.ticket.autoAssignment.button.cancel.label": "Отменить",
+  "pages.ticket.autoAssignment.button.refresh.label": "Обновить",
+  "pages.ticket.autoAssignment.button.save.label": "Сохранить",
+  "pages.ticket.autoAssignment.button.edit.label": "Редактировать",
+  "pages.ticket.autoAssignment.button.delete.label": "Удалить",
+  "pages.ticket.autoAssignment.notifications.save.message": "Data saved",
+  "pages.ticket.autoAssignment.notifications.delete.message": "Data deleted",
+  "pages.ticket.autoAssignment.notifications.error.message": "Failed to load data",
```
Comment (nomerdvadcatpyat) [inline apps/condo/lang/ru/ru.json:891]: en translations in ru file :)

File: apps/condo/pages/ticket/autoAssignment/index.tsx [hunk 3]
```diff
@@ -0,0 +1,574 @@
...
+            const values = await ClassifierLoader.search('', 'rules', null, 500)
+            setClassifiers(values)
+            setClassifierLoading(false)
+        })
+        return () => {
+            ClassifierLoader.clear()
+        }
+    }, [])
+
+    if (error) {
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/ticket/autoAssignment/index.tsx:456]: `|| loading` ?

File: apps/condo/pages/ticket/autoAssignment/index.tsx [hunk 4]
```diff
@@ -0,0 +1,574 @@
...
+    const canOpenPage = useMemo(
+        () => get(userWithRightSets, 'rightsSet.canManageTicketAutoAssignments')
+            && get(userWithRightSets, 'rightsSet.canReadTicketAutoAssignments'),
+        [userWithRightSets]
+    )
+
+    console.log({
+        data,
+        userWithRightSets,
+    })
```
Comment (nomerdvadcatpyat) [inline apps/condo/pages/ticket/autoAssignment/index.tsx:557]: remove log

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/lang/en/en.json (hunk 1 above)
Comment (Alllex202) [reply apps/condo/lang/en/en.json:891]: oops

File: apps/condo/pages/ticket/autoAssignment/index.tsx (hunk 3 above)
Comment (Alllex202) [reply apps/condo/pages/ticket/autoAssignment/index.tsx:456]: No, it's not necessary here

**Pull request merged**